### ISSUE TITLE
Multiple grids with animations

### ISSUE THREAD

**Issue by gielcobben** [akoenig/angular-deckgrid#71]
**Hi all,**

I have to pages, the blog page and the article page. Both of them have a Deckgrid. When I click on an article the ng-enter animation of the article page doesn't animate. I see Angular adding the right classes and styles for the animation but the article page just doesn't animate-in (ng-enter). When I remove the grid on the article page (so I have only one grid on the blog page) all the animations work perfect again.

Is there a way I can use Deckgrid on my blog page and on my article page with the animations?

Thanks,
**Giel.**

**Comment by anomer** [comment]
Did you find a fix or a workaround? I am stuck with the same issue.

**Comment by gielcobben** [comment]
Yes I have. Use z-index a higher z-index on the ng-enter than on the ng-leave.
